scanner/src/test/kotlin/LocalScannerTest.kt

```diff
@@ -85,27 +86,29 @@ private const val SCANNER_VERSION = "3.2.1.final"
 /**
  * Creates a [ScannerConfiguration] with the given properties for the test scanner.
  */
-private fun createConfig(properties: Map<String, String>): ScannerConfiguration {
+private fun createScannerConfig(properties: Map<String, String>): ScannerConfiguration {
     val options = mapOf(SCANNER_NAME to properties)
     return ScannerConfiguration(options = options)
 }
 
 /**
  * Create a test instance of [LocalScanner].
  */
-private fun createScanner(scannerConfig: ScannerConfiguration): LocalScanner =
-    object : LocalScanner(SCANNER_NAME, scannerConfig) {
-        override val configuration = "someConfig"
+private fun createScanner(
+    scannerConfig: ScannerConfiguration,
+    downloaderConfig: DownloaderConfiguration
+): LocalScanner = object : LocalScanner(SCANNER_NAME, scannerConfig, downloaderConfig) {
```

Comment:
Please fix the formatting; the newline should start after the `=`, and indentation of the following code should not change.